ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/ParticipantStateIntegrationSpecBase.scala

```diff
@@ -491,6 +491,9 @@ abstract class ParticipantStateIntegrationSpecBase(implementationName: String)
                   Duration.ofSeconds(123),
                   Duration.ofSeconds(123),
                   Duration.ofSeconds(123),
+                  Duration.ofSeconds(123),
+                  Duration.ofSeconds(123),
+                  Duration.ofSeconds(123),
```

Comment:
Perhaps pull the construction of the `TimeModel` out into a function called `aTimeModel` or something to keep the test case to the point?